legal/src/main/resources/legal/NOTICE.md

```diff
@@ -676,7 +676,7 @@ is also available at http://www.apache.org/licenses/LICENSE-2.0.html.
 
 The source code is available from [Maven Central](http://search.maven.org/remotecontent?filepath=io/vertx/vertx-web-client/${vertx.version}/vertx-web-client-${vertx.version}-sources.jar).
 
-### WildFly Common ${wildfly-common.version}
+### WildFly Common
```

Comment:
In this PR, _wildfly-common_  dependency has been removed from the _bom_. Is there any reason why we should still specify it here?

Reply:
Removing it from the BOM doesn't mean that it is no longer used. It is still pulled in by the Hot Rod client but we no longer explicitly manage its version. So FMPOV it is still necessary for us to specify it in the NOTICE file.